apps/web/__e2e__/models/settings-view.model.ts

```diff
@@ -59,4 +72,19 @@ export class SettingsViewModel {
   async isLoggedIn() {
     return await this.accountStatusContainer.isVisible();
   }
+
+  async createBackup(encryptBackups = false, password = "") {
+    await this.backupRestoreContainer.click();
+    if (encryptBackups) await this.encyptBackups.click();
+    await this.backupData.click();
+    if (encryptBackups) await fillPasswordDialog(this.page, password);
```

Comment:
Password dialog needs to be filled regardless of whether encryption is on or not.